### PR TITLE
added Quote and Order db to prisma.schema

### PR DESCRIPTION
**Pull request by rabanspiegel**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -58,4 +74,34 @@ model SupplierPart {
...
 
   @@id([supplierId, partId])
-}
\ No newline at end of file
+}
+
+model Quote {
+  id         Int      @id @default(autoincrement())
+  supplierId Int
+  partId     Int
```
Comment (bhausleitner) [inline prisma/schema.prisma:82]: `supplierId` and `partId` should not be in here, right?

File: prisma/schema.prisma [hunk 2]
```diff
@@ -58,4 +74,34 @@ model SupplierPart {
...
+
+  supplier   Supplier @relation(fields: [supplierId], references: [id])
+  part       Part     @relation(fields: [partId], references: [id])
+
+  @@index([status])
+}
+
+model Order {
+  id              Int      @id @default(autoincrement())
+  quoteId         Int
```
Comment (bhausleitner) [inline prisma/schema.prisma:98]: same here

File: prisma/schema.prisma (hunk 1 above)
Comment (rabanspiegel) [reply prisma/schema.prisma:82]: As far as I understand, they are correct to be there. 

I read this post to model my schema: https://www.prisma.io/docs/orm/prisma-schema/data-model/relations/one-to-many-relations

The 'part Part @relation(fields: [partId], references[id])' needs the partId field to make the link

File: prisma/schema.prisma (hunk 2 above)
Comment (rabanspiegel) [reply prisma/schema.prisma:98]: This is correct, I believe. This quoteId fields is referenced by the quote Quote to create the link. https://www.prisma.io/docs/orm/prisma-schema/data-model/relations/one-to-many-relations

**Review by bhausleitner — APPROVED** [review]

**Pull request merged**